viewers/wicket/ui/src/main/java/org/apache/isis/viewer/wicket/ui/components/bookmarkedpages/BookmarkedPagesPanel.html

```diff
@@ -25,7 +25,7 @@
 	<body>
         <wicket:panel>
             <div class="bookmarkedPagesPanel">
-                <div class="showPanelTab bookmarkRibbon bg-light ml-0"></div>
+                <div class="showPanelTab bookmarkRibbon bg-secondary ml-0"></div>
```

Comment:
in BS there was a class called navbar-inverted, but that has been removed

Reply:
We could invent our own "inverted" background colors, "bg-inverted", but we would have to do so for all themes we are supporting